src/holofuel/components/RecentCounterparties/RecentCounterparties.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
```

Comment:
Once we merge `CreateRequest` and `CreateOffer` into a single page, this component should be in that file, not it's own component.